exchanges/bitstamp/bitstamp_types.go

```diff
@@ -180,6 +180,29 @@ type CryptoWithdrawalResponse struct {
 	ID int64 `json:"withdrawal_id"`
 }
 
+// OpenBankWithdrawalRequest holds the request information for sending a bank withdrawal request
+type OpenBankWithdrawalRequest struct {
+	Amount         float64
+	Currency       string
```

Comment:
🤷 Considered shifting the responsibility down to `OpenBankWithdrawal` to call `.String()` ?
```suggestion
	Currency       currency.Code
```